pkg/template/registry/templateinstance/strategy.go

```diff
@@ -52,14 +52,9 @@ func (templateInstanceStrategy) Canonicalize(obj runtime.Object) {
 func (templateInstanceStrategy) PrepareForCreate(ctx apirequest.Context, obj runtime.Object) {
 	templateInstance := obj.(*templateapi.TemplateInstance)
 
-	//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, remove the if nil check and always
-	// Clear any user-specified info, then seed it from the context
-	if templateInstance.Spec.Requester == nil {
```

Comment:
Could you remind me on what basis this is being removed?  Does it affect the TSB setting the templateinstance requester on behalf of the TSB caller?  If so, a test should be tripped?

Reply:
Actually @jim-minter perhaps my TODO comment here was erroneous, given the test failure noted at https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/16434/test_pull_request_origin_extended_conformance_gce/8233/

Or maybe an associated change elsewhere is needed.

I've started investigating what I currently have here, and cycling through the original PR.  Will report back when I have discoveries of note.